releases/release-8.5.1.md

```diff
@@ -0,0 +1,103 @@
+---
+title: TiDB 8.5.1 Release Notes
+summary: Learn about the improvements and bug fixes in TiDB 8.5.1.
+---
+
+# TiDB 8.5.1 Release Notes
+
+Release date: January xx, 2025
+
+TiDB version: 8.5.1
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v8.1/quick-start-with-tidb) | [Production deployment](https://docs.pingcap.com/tidb/v8.1/production-deployment-using-tiup)
+
+## Operating system and platform requirement changes
+
+Starting from v8.5.1, TiDB resumes the support for CentOS Linux 7. If you plan to deploy TiDB v8.5 or upgrade your cluster to v8.5, ensure you use TiDB v8.5.1 or a later version.
```

Comment:
My suggestion would be:

> Starting from v8.5.1, TiDB resumed testing of CentOS Linux 7. If you plan to deploy TiDB v8.5 or upgrade your cluster to v8.5, ensure you use TiDB v8.5.1 or a later version.

Reply:
Does it mean that v8.5.1 is compatible with CentOS Linux 7 but does not provide official support for it?

Reply:
Yes, it was tested and works, but we are not adding it back to the list claiming we support it for production level quality - https://docs.pingcap.com/tidb/stable/hardware-and-software-requirements#os-and-platform-requirements